lib/utilities/concordia_constants.dart

```diff
@@ -19,21 +20,23 @@ const loyolaCampus = {
 
 const double poiZoomLevel = 17.5;
 const double campusZoomLevel = 16.5;
+const double navZoomLevel = 16.0;
 
-const buildings = {
+var buildings = {
```

Comment:
Terrific for adopting the Object based structure, using Campus and ConcordiaBuilding objects! Greatly approve of this